starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,11 @@
+body {
+background-color:white;
+max-width: 720px; 
+text-align: center;
+}
+
+.applePie {
```

Comment:
Try to use id's instead of classes. Just use classes if you want to reuse the styling in mere than one field